vertical-pod-autoscaler/enhancements/5030-vpa-manager/README.md

```diff
@@ -0,0 +1,338 @@
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+we want to make vpa more easier and automatically to use.
+
+### Goals
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+- use labels autoscaling/vpa=open or autoscaling/vpa=open to create or delete vpa or set some parameters for workloads in namespaces
+- use labels vpa/update-mode=initial or vpa/update-mode=auto to set pod update policy
+- use annotations in workloads to make it exceptional in namespaces, for example, when we want to create vpa for all workloads except deployment/test, we can annotate deployment/test to do that.
+- 
+
+### Non-Goals
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+- vpa manager don't manage all vpas, vpa manager just manager common and majority of workloads
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. What is the desired outcome and how do we measure success?.
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+### User Stories (Optional)
+
+<!--
+Detail the things that people will be able to do if this KEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+#### Story 1
+  when we want to create vpa for all workloads in namespace a, we just label namespace a with autoscaling/vpa=open, then vpa manager create vpa for every workload not annotated with autoscaling/vpa=close in namespace a, and later when workload are created in namespace a, corresponding vpa will be automaticlly created.
+
+  when we want to delete all vpas in namespace a, we just label namespace a with autoscaling/vpa=close, then vpa manager delete vpa for every workload not annotated with autoscaling/vpa=open in namespace a.
+
+#### Story 2
+
+  we can configure vpa manger default pod update mode as auto( or initial). when vpa manager create vpa, it will set vpa's updatepolicy.updateMode to auto (or initial). we can also specify update mode for namespace a by labeling namespace a with vpa/update-mode=initial( or vpa/update-mode=initial), in this case, all vpa created in namespace a has initial update mode except those vpa correspoding workloads annotated vpa/update-mode=auto.
+
+  later when we change update mode of namespace a by labeling it, all the vpa's update mode will be changed correspondingly in namespace a except those tarRef has been annotated by vpa/update-mode=***.
```

Comment:
So this means I can't change VPA mode by editing VPA, I need to change mode by adding a label, right?

Reply:
yes, it's for common use, if anyone want custom vpa parameters, he can annotate workloads with label "autoscaling/vpa=close" and create vpa manually